File: C#PartOne/Homeworks/03.Operators-and-Expressions-Homework/PrimeCheck/PrimeCheck.cs
namespace PrimeCheck
{
    using System;

    class PrimeCheck
    {
        static void Main(string[] args)
        {
            int number = int.Parse(Console.ReadLine());
            int template = (int)Math.Sqrt(number);
            bool isPrime = true;

            if (number < 2)
            {
                isPrime = false;
            }
            else
            {
                for (int i = 2; i <= template; i++)
                {
                    if (number % i == 0 && i != number)
                    {
                        isPrime = false;
                       
                    }

                }
            }


            if (isPrime)
            {
                Console.WriteLine("true");
            }
            else
            {
                Console.WriteLine("false");
            }
        }
    }
}

File: JavaScript-Fundamentals/Homeworks/04.Operators-and-Expressions/07. Prime Check/prime-check.js
function solve(args) {
    var number = +args,
        isPrime = true,
        cycleLimit = 0;

    if (number <= 2) {
        isPrime = false;
    }
    else {
        cycleLimit = Math.sqrt(number)
        for (var i = 2; i <= cycleLimit; i += 1) {
            if (number % i == 0 && i != number) {
                isPrime = false;
            }
        }
    }

    if (isPrime) {
        return true;
    }
    else {
        return false;
    }
}